client/components/domains/domain-registration-suggestion/index.jsx

```diff
@@ -87,6 +87,10 @@ class DomainRegistrationSuggestion extends React.Component {
 				resultSuffix = '#best-alternative';
 			}
 
+			if ( this.props.suggestion.is_premium ) {
+				resultSuffix = resultSuffix + '#premium';
```

Comment:
Super nitpick, but I prefer `resultSuffix += '#premium';`. Otherwise, I'm suspicious that the second `resultSuffix` is actually somehow different than the first one - since there must _have_ been a reason for not using the `+=` operator!
TL;DR: it's shorter, easier to parse (for human) and less prone to errors/typos 🙇

Reply:
Well - it turns our we can add a separate custom property so no need for a suffix :D